### PR TITLE
Resolve duplicate entries for መልክአ፡ ኪዳነ፡ ምሕረት፡

### PR DESCRIPTION
**Pull request by smaugustine**
See https://github.com/BetaMasaheft/Documentation/issues/1503

### CODE REVIEW

**Comment by eu-genia** [comment]
I must have copied the reference over from an EthioSPare record, as is

Zitat von Augustine Dickinson <notifications@github.com>:

> LIT1854Malkea had a reference to "Malkǝʾa gubāʾe, ʾAśmarā 1974 E.C.,  
> 112-122" with no other information. Zotero has an anonymous መልክአ፡  
> ጉባኤ፡ published in Asmara in 1948, but I suppose it's different? It's  
> difficult to tell with anonymous publications, esp. since there was  
> no other info given in LIT1854Malkea.
>
> --
> You are receiving this because your review was requested.
> Reply to this email directly or view it on GitHub:
> https://github.com/BetaMasaheft/Works/pull/311#issuecomment-702194006


-- 
Eugenia Sokolinski
Project Coordinator
Hiob Ludolf Centre for Ethiopian and Eritrean Studies
Asien Afrika Institut
Universität Hamburg
Alsterterrasse 1
20354 Hamburg
Tel. during the Home Office time +49 163 2879120
eugenia.sokolinski@uni-hamburg.de

https://www.aai.uni-hamburg.de/en/ethiostudies.html

**Comment by eu-genia** [comment]
@DenisNosnitsin1970 can you check the Malkǝʾa gubāʾe, ʾAśmarā 1974 E.C., 112-122 reference?

File: 3001-4000/LIT3082RepCh362.xml
Comment (eu-genia) [inline 3001-4000/LIT3082RepCh362.xml:2]: I would remove the t1 which is too general and misleading in search results (taken from Chaîne)
I leave it up to you whether t2 or t3 should become t1

File: 3001-4000/LIT3082RepCh362.xml
Comment (eu-genia) [inline 3001-4000/LIT3082RepCh362.xml:69]: Chaîne 362 should also be added to the bibliography

File: 3001-4000/LIT3082RepCh362.xml
Comment (eu-genia) [inline 3001-4000/LIT3082RepCh362.xml:78]: can be added to the editions, I have created the tag bm:MalkaGubae1974

**Review by eu-genia — CHANGES_REQUESTED** [review]
This PR can be only merged after all the MSS records have been adjusted

**Comment by DenisNosnitsin1970** [comment]
The reference to the edition is correct, I would propose that a couple more lines be added to the incipit. 
[Mälk'a Guba'e.pdf](https://github.com/BetaMasaheft/Works/files/5317462/Malk.a.Guba.e.pdf)

File: 3001-4000/LIT3082RepCh362.xml
Comment (smaugustine) [reply 3001-4000/LIT3082RepCh362.xml:2]: Yes, looking at it again I would definitely agree.

File: 3001-4000/LIT3082RepCh362.xml
Comment (eu-genia) [inline 3001-4000/LIT3082RepCh362.xml:73]: Denis was wishing a longer incipit eventually (checking also the book attached I guess, esp. in case it is any different). I will pre-approve but not merge, you can see if you merge or first add and then merge.

**Review by eu-genia — APPROVED** [review]

File: 3001-4000/LIT3082RepCh362.xml
Comment (smaugustine) [reply 3001-4000/LIT3082RepCh362.xml:73]: Thank you for the reminder! I've expanded the first incipit according to Dillmann's text.

**Pull request merged**
